source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,188 @@
+
+Specifying Array Elements
+-------------------------
+
+You can specify which array elements to update using a positional
+operator. Positional operators can specify the first, all or certain
+array elements to update. 
+
+To specify elements in an array with positional operators, use **dot
+notation**. Dot notation is a property access syntax for navigating JSON
+objects. 
+
+For additional information, see the server manual entry on
+:manual:`dot notation </core/document/#std-label-document-dot-notation>`.
+
+The First Array Element
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To update the first element that matches your query filter, use the
+positional ``$`` operator. The array field must appear as part of your
+query document to use the positional ``$`` operator. 
+
+Example
+```````
+
+The following example performs these actions:
+
+- Query for a document with a ``qty`` field containing the value "18"
+- Decrement the first array value that matches the query document by "3"  using the ``inc()`` method of the ``Updates`` builder and the positional ``$`` operator 
+- Update and retrieve the document
+
+.. literalinclude:: /includes/fundamentals/code-snippets/UpdateArray.java
+   :language: java
+   :dedent:
+   :start-after: begin updateValueExample
+   :end-before: end updateValueExample
+
+The example above updates the original document to the following state:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "color": "green", "qty": [8, 12, 15] }
+
+For additional information on the operator and method mentioned in this
+section, see the following resources: 
+
+- :manual:`Positional $ Operator </reference/operator/update/positional>` Server Manual Entry
+- :java-core-api:`inc() <com/mongodb/client/model/Updates.html#inc(java.lang.String,java.lang.Number)>` API Documentation
+
+All Array Elements
+~~~~~~~~~~~~~~~~~~
+
+To update all elements in an array, use the all positional ``$[]`` operator. 
+
+Example
+```````
+
+The following example performs these actions: 
+
+- Query for the sample document
+- Multiply array elements matching the filter by "2"  using the ``mul()`` method of the ``Updates`` builder and the all ``$[]`` positional operator
+- Update and retrieve the document
+
+.. literalinclude:: /includes/fundamentals/code-snippets/UpdateArray.java
+   :language: java
+   :dedent:
+   :start-after: begin updateAllElementsExample
+   :end-before: end updateAllElementsExample
+
+The example above updates the original document to the following state:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "color": "green", "qty": [16, 24, 36] }
+
+For additional information on the operator and method mentioned in this
+section, see the following resources: 
+
+- :manual:`All Positional $[] Operator </reference/operator/update/positional-all/>` Server Manual Entry
+- :java-core-api:`mul() <com/mongodb/client/model/Updates.html#mul(java.lang.String,java.lang.Number)>` API Documentation
+
+Multiple Array Elements
```

Comment:
```suggestion
Array Elements Matching a Filter
```

Suggestion: I think the title "Multiple Array Elements" does not communicate the content of the section (matching with a filter). I think mentioning "Filter" in the title could better communicate the section's content.